static/css/components/cbox.less

```diff
@@ -22,9 +22,7 @@
 #cbox {
   &Overlay {
     position: fixed;
-    width: 100%;
-    height: 100%;
-    background: @brown-3a301e;
+    background: @grey; //change color here
```

Comment:
This color appears in a few places throughout this file. What do you think about creating a new variable for it? That way if we have to change it again in the future, we have to change it only in one place.